src/org/usfirst/frc/team467/robot/Robot.java

```diff
@@ -132,16 +137,32 @@ public void testPeriodic() {
 	}
 
 	public void autonomousPeriodic() {
-		autonomous.run();
+//		autonomous.run();
 	}
 
 	/**
 	 * This function is called periodically during operator control
 	 */
 	public void teleopPeriodic() {
+		//TODO: Set Min_DRIVE_SPEED in Robot Map.
+		double MIN_DRIVE_SPEED = 0.1;
+		driverstation.readInputs();
+		XboxController xbox = new XboxController(0);
+		double left = driverstation.getDriveJoystick().getLeftStickY()*-1;
+		double right = driverstation.getDriveJoystick().getRightStickX();
```

Comment:
We should clean this up a bit more, but it works for now.

Reply:
I thought you guys changed this to use the DriverStation class.